website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -980,24 +989,39 @@ as shown in commands like `nomad server members`. To see the version of the Raft
 protocol in use on each server, use the `nomad operator raft list-peers`
 command.
 
-The easiest way to upgrade servers is to have each server leave the cluster,
-upgrade its `raft_protocol` version in the `server` stanza, and then add it
-back. Make sure the new server joins successfully and that the cluster is stable
-before rolling the upgrade forward to the next server. It's also possible to
-stand up a new set of servers, and then slowly stand down each of the older
-servers in a similar fashion.
-
 When using Raft protocol version 3, servers are identified by their `node-id`
 instead of their IP address when Nomad makes changes to its internal Raft quorum
 configuration. This means that once a cluster has been upgraded with servers all
 running Raft protocol version 3, it will no longer allow servers running any
-older Raft protocol versions to be added. If running a single Nomad server,
-restarting it in-place will result in that server not being able to elect itself
-as a leader. To avoid this, either set the Raft protocol back to 2, or use
-[Manual Recovery Using
+older Raft protocol versions to be added.
+
+~> **Warning:** If you are running a single Nomad server, restarting it
+in-place will result in that server not being able to elect itself as
+a leader. To avoid this, either set the Raft protocol back to 2, or
+use [Manual Recovery Using
 peers.json](https://learn.hashicorp.com/tutorials/nomad/outage-recovery#manual-recovery-using-peersjson)
 to map the server to its node ID in the Raft quorum configuration.
 
+The easiest way to upgrade servers is to have each server leave the cluster,
+upgrade its [`raft_protocol`] version in the `server` stanza, and then add it
+back. Make sure the new server joins successfully and that the cluster is stable
+before rolling the upgrade forward to the next server. It's also possible to
+stand up a new set of servers, and then slowly stand down each of the older
+servers in a similar fashion.
+
+For in-place raft protocol upgrades, perform the following for each server:
```

Comment:
Would it be worth noting when the leader should be updated? Usually I'll leave it last to avoid leadership changes during updates which may also be useful here?

Reply:
That's probably a good idea.